src/firefly/js/visualize/ImagePlotCntlr.js

```diff
@@ -84,7 +84,22 @@ const CHANGE_EXPANDED_MODE= 'ImagePlotCntlr.changeExpandedMode';
 const EXPANDED_AUTO_PLAY= 'ImagePlotCntlr.expandedAutoPlay';
 const EXPANDED_LIST= 'ImagePlotCntlr.expandedList';
 
-const CHANGE_MOUSE_READOUT_MODE= 'ImagePlotCntlr.changeMouseReadoutMode';
+//LZ add those three constant on 2/1/16
+//const CHANGE_MOUSE_READOUT_READOUT1 =  ()=> {
+//    return {
+//      type: CoordinateSys,
+//      text:'ImagePlotCntlr.changeMouseReadoutModeReadout1'
+//   };
+//} ;
+//const CHANGE_MOUSE_READOUT_READOUT2= ()=>{
+//    return {
+//        type: CoordinateSys,
+//        text: 'ImagePlotCntlr.changeMouseReadoutModeReadout2'
+//    };
+//};
+const CHANGE_MOUSE_READOUT_READOUT1='ImagePlotCntlr.changeMouseReadoutModeReadout1';
+const CHANGE_MOUSE_READOUT_READOUT2='ImagePlotCntlr.changeMouseReadoutModeReadout2';
+const CHANGE_MOUSE_READOUT_PIXEL= 'ImagePlotCntlr.changeMouseReadoutModeReadoutPixel';
```

Comment:
I don't think I clearly communicated about the actions.  I saw you changed the 3 dispatcher into one.  However, I think you should make the action ID into one.  Let the parameters decide which of the three are changed. In the dispatcher you pass a type. Just keeping sending this type all the way through into the reducer.  Then let the reducer do the correct update.

We can talk about this some more if you want.